cli/command/trust/key_load.go

```diff
@@ -82,9 +87,29 @@ func getPrivKeyBytesFromPath(keyPath string) ([]byte, error) {
 }
 
 func loadPrivKeyBytesToStore(privKeyBytes []byte, privKeyImporters []trustmanager.Importer, keyPath, keyName string, passRet notary.PassRetriever) error {
-	if _, _, err := tufutils.ExtractPrivateKeyAttributes(privKeyBytes); err != nil {
+	var err error
+	if _, _, err = tufutils.ExtractPrivateKeyAttributes(privKeyBytes); err != nil {
 		return fmt.Errorf("provided file %s is not a supported private key - to add a signer's public key use docker trust signer add", keyPath)
 	}
+	if privKeyBytes, err = decodePrivKeyIfNecessary(privKeyBytes, passRet); err != nil {
+		return errors.Wrapf(err, "cannot load key from provided file %s", keyPath)
+	}
 	// Make a reader, rewind the file pointer
 	return trustmanager.ImportKeys(bytes.NewReader(privKeyBytes), privKeyImporters, keyName, "", passRet)
 }
+
+func decodePrivKeyIfNecessary(privPemBytes []byte, passRet notary.PassRetriever) ([]byte, error) {
```

Comment:
Would be good to have a unit-test for this one, but I'm okay adding that in a follow-up